api/logs/v1/rules_prometheus_labels_enforcer.go

```diff
@@ -0,0 +1,280 @@
+package http
+
+import (
+	"bytes"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"io"
+	"net/http"
+	"strconv"
+	"time"
+
+	"github.com/go-kit/log"
+	"github.com/go-kit/log/level"
+	"github.com/observatorium/api/authentication"
+	"github.com/observatorium/api/authorization"
+	"github.com/prometheus/prometheus/model/labels"
+)
+
+const contentTypeApplicationJSON = "application/json"
+
+var (
+	errUnknownTenantKey        = errors.New("Unknown tenant key")
+	errUnknownRulesContentType = errors.New("Unknown rules response content type")
```

Comment:
Make the error message start lower-case?

```suggestion
	errUnknownTenantKey        = errors.New("unknown tenant key")
	errUnknownRulesContentType = errors.New("unknown rules response content type")
```